pylintrc

```diff
@@ -40,7 +40,7 @@ load-plugins=
 
 disable=R, missing-docstring, fixme, invalid-name, bad-continuation, attribute-defined-outside-init, arguments-differ,
     attribute-defined-outside-init, star-args, locally-disabled, unused-argument, unused-variable, no-init,
-    global-statement,
+    global-statement, bad-builtin
```

Comment:
It looks like the use of `map` is discouraged in Python 3. Is there a reason you are not using a list comprehension instead?

http://www.artima.com/weblogs/viewpost.jsp?thread=98196
https://docs.python.org/3.1/whatsnew/3.0.html#views-and-iterators-instead-of-lists